Expensify/App
# Added tooltip sense & add missing tooltips

### Details

- [x] Instant Tooltip.
- [x] Add missing tooltips.

### Fixed Issues

$ https://github.com/Expensify/App/issues/5347

### Tests | QA Steps
1. Open NewDot on Web.
2. hover over any target that has a tooltip.
3. Now move your cursor to another tooltip target within 1 sec.
4. Tooltip should be shown immediately.
5. Change multiple targets and the same behavior should be shown.
6. Now move your cursor from tooltip target to an area that does not have any tooltip attached.
7. Wait for 1 sec, move back your cursor over the tooltip target.
8. Tooltip should show with animation and delay.
9. Perform repeat steps 3-8 to observe the changed behavior (Instant tooltips).

### Tested On

- [ ] Mobile Web (does not have tooltips)
- [ ] iOS (does not have tooltips)
- [ ] Android (does not have tooltips)

### Screenshots


#### Web



https://user-images.githubusercontent.com/24370807/136609468-3c5fb333-573f-45bd-a4a4-26660eea154f.mp4



#### Mobile Web


#### Desktop


#### iOS


#### Android
